src/idsk/components/customer-surveys/customer-surveys.js

```diff
@@ -0,0 +1,212 @@
+import "../../../govuk/vendor/polyfills/Function/prototype/bind";
+import "../../../govuk/vendor/polyfills/Event"; 
+import { toggleClass } from "../../common";
+
+function CustomerSurveys($module) {
+    this.$module = $module;
+}
+
+CustomerSurveys.prototype.init = function () {
+    var $module = this.$module;
+    var $nextButton = $module.querySelector('#idsk-customer-surveys__send-button');
+    var $previousButton = $module.querySelector('#idsk-customer-surveys__previous-button');
+    var $textAreaFirst = $module.querySelector('.idsk-customer-surveys-text-area #first');
+    var $textAreaSecond = $module.querySelector('.idsk-customer-surveys-text-area #second');
+    var $textAreaThird = $module.querySelector('.idsk-customer-surveys-text-area #third');
+    var $textAreaFourth = $module.querySelector('.idsk-customer-surveys-text-area #fourth');
+    var $radioButtonWork = $module.querySelector('.idsk-customer-survey__radio--work');
+    var $radioButtonPrivate = $module.querySelector('.idsk-customer-survey__radio--private');
+    var $counter = 7;
+    $module.sendButtonDisabled = new Array(7);
```

Comment:
staci len "$module.sendButtonDisabled = []" netreba davat new Array

Reply:
toto by som radsej nechal tak lebo aj tak budem musiet niekde povedat ze to ma mat 7 itemov, napr. vo fore.

Reply:
oki, moze buc :)